Polaris/src/SensorBoardLibraries/SensorBoard.hpp

```diff
@@ -64,6 +64,13 @@ class Sensorboard{
 
         // mag.readSensor(*magData);
         this->ProcessBuffer();
+
```

Comment:
By the way the change here is the data structure is slightly different from rocket.